Bug 1123325 - PasscodeHelper in FindMyDevice (2 out of N)

mozfreddyb opened the pull request:
Next patch, for bug 1123325: Change FMD to use PasscodeHelper

mozfreddyb committed:
Bug 1123325 - PasscodeHelper in FindMyDevice r=mgoodwin

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e5043914e34e9c5ce4e75937a7fc3df37599d29

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05405f95f5572c93d0237b77f06ddec74fb6bb1b

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c71735fb65237be50481c6420dc2ef30b75e0e79

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9f9ff434b81f2a1e1bcc1195f71b26781fefeba8

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4e35d2a88d5fec0d04569a0f30fea8fd37eefae8

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c914e3af91de6a218921480e14374817d3011624

mozfreddyb commented:
Github somehow lost @mozmark's comments on the previous commit, but I just addressed the nits (mostly variable naming and whitespaces) in the most recent commit.

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3592dc10fbe44028be962716b35c9c10d08985c4

mozfreddyb commented:
rebased to master

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5181df972bb4951bb7997c085cf2fb7aa1ab6c8e
